docs/sources/installation/windows.md

```diff
@@ -43,7 +43,3 @@ Read more about the [configuration options]({{< relref "configuration.md" >}}).
 The Grafana backend includes Sqlite3 which requires GCC to compile. So
 in order to compile Grafana on Windows you need to install GCC. We
 recommend [TDM-GCC](http://tdm-gcc.tdragon.net/download).
-
-Copy `conf/sample.ini` to a file named `conf/custom.ini` and change the
```

Comment:
Why removing this?

Reply:
Because it is already mentioned in a previous paragraph (line 26 , https://github.com/grafana/grafana/pull/11031/files/7d91b737749fa0293a53fbc115ad798ad26c1ab3#diff-69d8cc6ef27703bcb60976e3f377c820L26 )

Reply:
Gotcha :+1: